expirationd.lua

```diff
@@ -192,12 +183,17 @@ end
 -- Task management
 -- ------------------------------------------------------------------------- --
 
+--[[
+task:start() - Force start `task` (old `guardian_fiber` will be garbage
+               collected, so do `task:stop()` before or `task:restart()` instead)
```

Comment:
I know that the documentation for task: api is not ready yet. But I will comment on what you have already written.
> old `guardian_fiber` will be garbage collected, so do` task:stop()` before or` task:restart()` instead

I did not understand the point, it also calls `task:stop` inside and waits for the previous task to stop.

Reply:
> I know that the documentation for task: api is not ready yet. 

This PR is not about changing API, but about documenting it right and precisely.

> But I will comment on what you have already written.

It's text that was written before in comments, I just formatted it as LDoc comment. However I'll check it this still valid.

Reply:
Removed.